### ISSUE TITLE
scExpression: add organism info to tissue and genes

### ISSUE THREAD

**Issue by signechambers1** [chanzuckerberg/single-cell-data-portal#2344]
The user experience we are aiming for is that when a user selects an organism the tissue and genes update to reflect what is available for that organism. For this to work, the front end needs to know which tissues and genes are associated with which organism.

This ticket covers the back end work to add organism to the API info for tissues and genes.

**Comment by MDunitz** [comment]
@tihuan do you want to add  
- [ ] the current endpoint response
- [ ] the proposed format that would allow the front end to do this filtering?

**Comment by tihuan** [comment]
Yeah genes are keyed by organisms in primary filters already, we just need to do the same thing for tissues please 🙆‍♂️

**Comment by tihuan** [comment]
Updated the ticket description 🙏 Thanks so much, @MDunitz 🎉 !

**Comment by atolopko-czi** [comment]
AFICT, this story is a dup of #2337, which is already implemented. OK to close this one now?
